deployment/README.md

```diff
@@ -1,72 +1,129 @@
-
-<!-- markdownlint-disable MD033 -->
-<p align="center">
-  <a href="https://ocelot.social" target="_blank"><img src="branding/static/img/custom/logo-squared.svg" alt="Ocelot-Social" width="40%" height="40%"></a>
-</p>
-<!-- markdownlint-enable MD033 -->
+The `deployment` directory is about how to change the look and feel of ocelot.social (called branding) and how to get the software running on a kubernetes cluster (deployment).
 
 ## Live demo
 
 __Try out our deployed [development environment](https://stage.ocelot.social).__
 
-Visit our staging networks:
-
-- central staging network: [stage.ocelot.social](https://stage.ocelot.social)
-<!-- - rebranded staging network: [rebrand.ocelot.social](https://stage.ocelot.social). -->
+This environment is the vanilla ocelot without branding and with seed data.
 
-Logins:
+### Logins
 
 | email | password | role |
 | :--- | :--- | :--- |
 | `user@example.org` | 1234 | user |
 | `moderator@example.org` | 1234 | moderator |
 | `admin@example.org` | 1234 | admin |
 
+## Software requirements
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [helm](https://helm.sh/docs/intro/install/)
+
+Depending on your Kubernetes providers you deploy on:
+| Cluster | Type | Tool |
+| DigitalOcean | Cloud Hosting | [doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/) |
+| Minicube | Local Deployment | [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) |
+
+If you're just getting started, Minikube is a tool that you can use to get your feet wet.
+
 ## Usage
 
-Fork this repository to configure and rebrand it for your own [ocelot.social](https://github.com/Ocelot-Social-Community/Ocelot-Social) network.
+The following tasks can be achieved with the tools contained in this directory:
 
-### Package.Json And DockerHub Organisation
+### Start branded docker environment locally
 
-Write your own data into the main configuration file:
+For the developers we offer a `docker-compose` in this directory.
 
-- [package.json](/package.json)
+1. Copy `.env.dist` and rename it to `.env`
+2. (optional) change the `CONFIGURATION` in the `.env` value according to your desired brand. By default its `example`
+3. Run `docker-compose up`
 
-Since all deployment methods described here depend on [Docker](https://docker.com) and [DockerHub](https://hub.docker.com), you need to create your own organisation on DockerHub and put its name in the [package.json](/package.json) file as your `dockerOrganisation`.
+This will start the ocelot services within docker in the selected branded version.
 
-### Configure And Branding
+If you stop the docker-compose, change the value of `CONFIGURATION` in the `.env` and restart with `docker-compose up` the branded version will change.
 
-The next step is:
+Note: this might require `export DOCKER_BUILDKIT=0` to be set in order to have docker-compose build correctly.
```

Comment:
```suggestion
Note: this might require `export DOCKER_BUILDKIT=0` to be set in order to have `docker compose` build correctly.
```

Or as brand:

```suggestion
Note: this might require `export DOCKER_BUILDKIT=0` to be set in order to have Docker Compose build correctly.
```